content/applications/finance/accounting/fiscal_localizations/overview/fiscal_localization_packages.rst

```diff
@@ -22,10 +22,6 @@ selected at the creation of the database.
 To install a new package, go to :menuselection:`Accounting --> Configuration --> Fiscal
 Localization`, click on **Install More Packages**, and install your country's module.
 
-.. image:: media/fiscal_localization_packages_modules.png
-   :align: center
-   :alt: Install the appropriate module as fiscal localization package in Odoo Accounting.
-
 Once done, select your country's package, and click on *Save*.
 
 .. image:: media/fiscal_localization_packages_selection.png
```

Comment:
You deleted the image but did not remove references to that image, hence the red runbot build.

Reply:
Ah! Silly mistake. Thanks for catching.
Should be resolved on latest commit [931710b](https://github.com/odoo/documentation/pull/1311/commits/931710b52b063e3d6a853713dbdb8d766d406f76). Runbot built passed.